checker.c

```diff
@@ -22,8 +22,7 @@
 
 bool checkBattParameter(float VALUEIN, float Const_Min , float Const_Max)
 {
-	int BattParamValue = ((Const_Min < VALUEIN) || (VALUEIN <= Const_Max));
-	return (BattParamValue);
+	return ((Const_Min < VALUEIN) || (VALUEIN <= Const_Max));
```

Comment:
Valid test cases are failing 
Particularly SOC 
i am not seeing why it is so.

Reply:
Maybe the condition must be &&
Otherwise it will return true whenever Const_Min is less than the value passed - even if it is greater than Const_Max